spec/suma/api/commerce_spec.rb

```diff
@@ -272,10 +312,30 @@
       expect(checkout.refresh).to be_completed
     end
 
+    it "errors if fulfillment option id is missing" do
```

Comment:
Do not write tests for `requires` vs. `optional`, and other trivial settings in the `params` block.